pkg/kustomize/secrets.go

```diff
@@ -229,6 +232,22 @@ func ContainsComponentConfig(
 		fields = (&securityscanner.SecurityScannerFieldGroup{}).Fields()
 	case v1.ComponentPostgres:
 		fields = (&database.DatabaseFieldGroup{}).Fields()
+	case v1.ComponentClairPostgres:
+		clairConfigBytes, ok := configBundle["clair-config.yaml"]
+		// Clair config not provided
+		if !ok {
+			return false, nil
+		} else {
+			var clairConfig config.Config
+			if err := yaml.Unmarshal(clairConfigBytes, &clairConfig); err != nil {
+				return false, err
+			}
+			// Else check if connstring is provided anywhere in the config
+			if clairConfig.Matcher.ConnString != "" || clairConfig.Indexer.ConnString != "" || clairConfig.Notifier.ConnString != "" {
```

Comment:
would you mind breaking this line? I don't like long lines, it is quite disturbing. wdyt?